src/utilities/url-params.ts

```diff
@@ -7,13 +7,15 @@ export interface QueryParams extends ConnectedBioAuthoring {
 let params: QueryParams;
 
 try {
-  const queryString = location.search.length > 1 ? decodeURIComponent(location.search.substring(1)) : "{}";
+  const queryString = location.search.length > 1
+    ? decodeURIComponent(location.search.substring(1).replace("authoring", ""))
+    : "{}";
   params = JSON.parse(queryString);
 } catch (e) {
   params = {};
 }
 
-if (location.search === "?authoring") {
+if (location.search.indexOf("authoring") > -1) {
```

Comment:
I don't expect this to happen, but if the string "authoring" appears in the instructions, then we'll reroute back to the authoring page when the user hits submit.

Reply:
Yeah, I know. I went back and forth between being worried about that, even though the string doesn't currently appear in the authoring spec, and thinking that folks might put it after the `%` in `?%7B"curriculum"%3A"mouse"....`, and wanting to still support them if they did that.

But I didn't think about the remote possibility that someone might author instructions with the word "authoring" in it. It's a slim chance, but a good point. I'll fix it so that `authoring` must come right at the beginning.